images/ansible-operator/base.Dockerfile

```diff
@@ -2,7 +2,7 @@
 # It is built with dependencies that take a while to download, thus speeding
 # up ansible deploy jobs.
 
-FROM registry.access.redhat.com/ubi8/ubi:8.3-297
+FROM registry.access.redhat.com/ubi8/ubi:8.3-297.1618432833
```

Comment:
It this right? the others have -298. this has -297.

Reply:
ubi vs ubi-minimal